src/auditor/compiler_abi.jl

```diff
@@ -185,7 +186,7 @@ function check_cxxstring_abi(oh::ObjectHandle, platform::Platform; io::IO = stdo
         both GCC 4 and GCC 5.  To do this, immediately after your `products`
         definition in your `build_tarballs.jl` file, add the line:
         """, '\n' => ' '))
-        msg *= "\n\n    products = expand_cxx_versions(products)"
+        msg *= "\n\n    products = expand_cxxstring_abis(products)"
```

Comment:
The whole message is wrong, not just this line.  `expand_cxxstring_abis` applies to platforms, not products:
```suggestion
        msg *= "\n\n    platforms = expand_cxxstring_abis(platform)"
```
See https://github.com/JuliaPackaging/Yggdrasil/pull/393#discussion_r367450854